imports/api/accounts/server/methods.ts

```diff
@@ -107,7 +107,7 @@ Meteor.methods({
     account.balance = balance;
     account.totalBalance = data;
 
-    if (parseInt(balance) > 0) {
+    if (parseInt(balance) >= 0) {
```

Comment:
Does it mean the balance of the account can be 0 after it has just been created?

Reply:
It will update accounts that have 0 CELO balance (Some accounts have 0 CELO but have locked CELO and until now those accounts were not updated/stored in DB)